DependencyInjection/SonataAdminExtension.php

```diff
@@ -96,7 +96,38 @@ public function load(array $configs, ContainerBuilder $container)
         }
 
         $container->setParameter('sonata.admin.configuration.global_search.empty_boxes', $config['global_search']['empty_boxes']);
-        $container->setParameter('sonata.admin.configuration.templates', $config['templates']);
+        $container->setParameter('sonata.admin.configuration.templates', $config['templates'] + array(
```

Comment:
@sonata-project/contributors technically this is a BC-break but I think it is ok to assume no one is relying on it.